src/components/facet/_facet_group.scss

```
@@ -1,9 +1,23 @@
+$gutterTypes: (
```

SASS Maps and variables need to be declared in a `_variables.scss` file and imported by the sibling `_index.scss` file and the names need to be more unique.
```suggestion
$facetGroupGutterTypes: (
```